src/components/SpdxPicker.js

```diff
@@ -39,6 +39,7 @@ export default class SpdxPicker extends Component {
     return (
       <div className="editable-editor">
         <Typeahead
+          className="spdx-input-picker"
```

Comment:
`id`?

Reply:
Applying an `id` is not possible in this case.
The `SpdxPicker` is used multiple times in the application (like in the `LicensePicker` for example) so an id would generate some conflicts in the DOM.